nodes/src/nodes/prompt_template/IInstance.py

```
@@ -0,0 +1,84 @@
+            template = config.get('template', '{{input}}')
+            context = self._get_template_context()
+
+            # If we have questions, output them
+            if self.question.questions:
+                self.instance.writeQuestions(self.question)
+            # If we only have text input, render and output as text
+            elif self.collected_text:
+                rendered = render(template, context)
+                self.instance.writeText(rendered)
+
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)